services/cd-service/pkg/repository/transformer.go

```diff
@@ -34,6 +32,9 @@ import (
 	"strings"
 	"time"
 
+	time2 "github.com/freiheit-com/kuberpult/pkg/time"
```

Comment:
please fix the typo in the PR title ("environment")
And explain the root cause of the issue in the description.